matrix-vis.js

```diff
@@ -105,11 +105,24 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
+        var Type = " ";
+        var Sender = " ";
+        var m = " ";
         if (params.nodes.length === 1) {
-            text = JSON.stringify(nodes.get(params.nodes[0]).blob, undefined, 2);
+            Type=JSON.stringify(nodes.get(params.nodes[0]).blob.type, undefined, 2);
+            Sender=JSON.stringify(nodes.get(params.nodes[0]).blob.sender, undefined, 2);
+            t1="Event Type : "+Type;
+            t2="Creator : "+Sender;
+            if (Type==JSON.stringify("m.room.member")) {
+            m=JSON.stringify(nodes.get(params.nodes[0]).blob.content.membership, undefined, 2);
+            t3="Action : "+m;
+            }
```

Comment:
**Indentation!**

You MUST indent when you write code! It makes it *a lot* harder to read otherwise.